tailwind.config.js

```diff
@@ -16,6 +16,17 @@ const fontSizes = {
   '5xl': '2.5rem',
 };
 module.exports = {
+  purge: {
+    enabled: process.env.NODE_ENV === 'production',
+    content: [
+      './components/**/*.vue',
+      './components/*.vue',
+      './components/*.js',
+      './stories/**/*.mdx',
+      './stories/**/**/*.mdx',
```

Comment:
I don't think we should include these - as tailwind classes used here shouldn't be part of our bundle.